packages/core/src/messagingProtocols/mls/MLSService/MLSService.ts

```diff
@@ -66,23 +66,33 @@ import {
 import {ClientId, HandlePendingProposalsParams} from '../types';
 import {generateMLSDeviceId} from '../utils/MLSId';
 
+type Optional<T, K extends keyof T> = Pick<Partial<T>, K> & Omit<T, K>;
+
+interface MLSConfig {
+  ciphersuites: Ciphersuite[];
+  defaultCiphersuite: Ciphersuite;
+  /**
+   * (milliseconds) period of time between automatic updates of the keying material (30 days by default)
+   */
+  keyingMaterialUpdateThreshold: number;
+  /**
+   * number of key packages client should upload to the server (100 by default)
+   */
+  nbKeyPackages: number;
+}
+export type InitClientOptions = Optional<MLSConfig, 'keyingMaterialUpdateThreshold' | 'nbKeyPackages'> & {
+  skipInitIdentity?: boolean;
+};
+
 //@todo: this function is temporary, we wait for the update from core-crypto side
 //they are returning regular array instead of Uint8Array for commit and welcome messages
 export const optionalToUint8Array = (array: Uint8Array | []): Uint8Array => {
   return Array.isArray(array) ? Uint8Array.from(array) : array;
 };
 
-interface LocalMLSServiceConfig extends MLSServiceConfig {
-  /**
-   * minimum number of key packages client should have available (configured to half of nbKeyPackages)
-   */
-  minRequiredNumberOfAvailableKeyPackages: number;
-}
-
-const defaultConfig: MLSServiceConfig = {
+const defaultConfig = {
   keyingMaterialUpdateThreshold: 1000 * 60 * 60 * 24 * 30, //30 days
   nbKeyPackages: 100,
-  cipherSuite: Ciphersuite.MLS_128_DHKEMX25519_AES128GCM_SHA256_Ed25519,
```

Comment:
we don't provide default ciphersuite, the consumer needs to manually set the suite they want to use